docs/embeddings/configuration/cloud.md

```diff
@@ -26,21 +26,30 @@ Container/bucket/directory/repository name.
 
 ## Cloud object storage configuration
 
-In addition to the above common configuration, the cloud object storage provider has the following additional configuration parameters.
+In addition to the above common configuration, the cloud object storage provider has the following additional configuration parameters. The number of these configs which are need depend upon your cloud provider. Some cloud providers do not need any of these configurations and can use implicit authorization with service accounts.
+
+These parameters are defined in the [libcloud documentation](https://libcloud.readthedocs.io/en/stable/apidocs/libcloud.common.html#module-libcloud.common.base).
 
 ### key
 ```yaml
 key: string
 ```
 
-Provider-specific access key. Can also be set via ACCESS_KEY environment variable. Ensure the configuration file is secured if added to the file.
+Optional provider-specific access key. Can also be set via `ACCESS_KEY` environment variable. Ensure the configuration file is secured if added to the file.
 
 ### secret
 ```yaml
 secret: string
 ```
 
-Provider-specific access secret. Can also be set via ACCESS_SECRET environment variable. Ensure the configuration file is secured if added to the file.
+Optional provider-specific access secret. Can also be set via `ACCESS_SECRET` environment variable. Ensure the configuration file is secured if added to the file.
```

Comment:
Why would we want to enable this option? I think it's better from a security standpoint to not even give the option for HTTP.

Reply:
I'm generally of the opinion that when implementing a library or framework, we should provide good defaults (like defaulting to HTTPS), but that we shouldn't take choices away from users. I included this because it is [part of the BaseProvider](https://github.com/apache/libcloud/blob/6e39117d6f686e2f49b12fc8a24168d13b6e0ada/libcloud/common/base.py#L1088-L1111).

Reply:
I'm trying to think of a use case for this but can't.

From looking at the libcloud GitHub, this seems like a legacy parameter from the mid 2010s before we moved to defaulting services to HTTPS.

Unless someone has a use case or is asking for it, I'd prefer to just always default to `secure=True` like it does currently.